package-examples/mysql-kustomize/README.md

```diff
@@ -0,0 +1,86 @@
+# mysql-kustomize
+
+This is a package that has several hydration functions in it's pipeline
+as well as utilzing kustomize to patch an upstream configuration.  The 
+upstream configuration is saved in the /upstream folder and the /instance
+patch is configured using kpt functions.
+
+For the final configuration kustomize is used to build the patched config
+with `kpt live apply` doing the deployment.
```

Comment:
I really have not idea of mixing this `kustomize` and `kpt` pkg will do :) I know the intention is to migrate and ensure syntax is correct. But in this case, it feels like we will be better off by deleting this example and introduce it later to help user accomplish some particular task. This can get really confusing.

Reply:
Sounds good.  I will remove the sample!

Reply:
Thanks.